shared/js/background/classes/tab-state.js

```diff
@@ -32,6 +32,8 @@ export class TabState {
         this.trackers = {}
         /** @type {null | import('../events/referrer-trimming').Referrer} */
         this.referrer = null
+        /** @type {boolean} */
+        this.ctlYouTubeAdded = false
```

Comment:
Not quite clear what this variable represents, perhaps we could rename it to make it more obvious? (Is it to say that a CTL YouTube placeholder was shown for the page?)

Reply:
Hmm, maybe just call it `ctlYouTube`? That seems more consistent with the other similar names above.

Maybe add a comment here to explain though? E.g. "True when at least one YouTube Click to Load placeholder was displayed for the tab."